proto/internalapi/sensor/admission_control.proto

```diff
@@ -21,6 +21,7 @@ message AdmissionControlSettings {
 
   string cluster_id = 6;
   storage.PolicyList runtime_policies = 7;
+  bool flatten_image_data_on_central = 8;
```

Comment:
nit: is there a better name for this field, specifially '_on_central" - first reaction was: this somehow triggers a request to central to flatten image data.